.github/dependabot.yml

```
@@ -1,11 +1,24 @@
 version: 2
 updates:
```

Shouldn't we add some directive about GitHub actions? On other projects I maintain we have something like that:

```suggestion
updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
    labels:
      - "area/dependencies"
```

---

I feel so stupid, I've missed the GH action section